src/App/Components/Common/NavProfile/navbar.js

```diff
@@ -17,24 +42,22 @@ export class navbar extends Component {
           aria-controls="navbarNavAltMarkup"
           aria-expanded="false"
           aria-label="Toggle navigation"
+          onMouseOver={this.hideNotificationsComponent}
         >
-          <span className="navbar-toggler-icon" />
+          <i className="fas fa-bars" />
```

Comment:
@nkpremices is `fas` a valid class name or is this an error?

Reply:
This is `fontAwesome` @MemunaHaruna